Source/WTF/wtf/glib/ActivityObserver.h

```diff
@@ -87,7 +97,8 @@ class ActivityObserver : public ThreadSafeRefCounted<ActivityObserver> {
     ThreadSafeWeakPtr<RunLoop> m_runLoop;
     uint8_t m_order { 0 };
     OptionSet<RunLoop::Activity> m_activities;
-    Callback m_callback;
+    mutable Lock m_callbackLock;
+    Callback m_callback WTF_GUARDED_BY_LOCK(m_callbackLock);
```

Comment:
Why does only the callback need to be locked? What about `m_order` and `m_activities`?

Reply:
They never change -- immutable after destruction.